test/functional/apps/visualize/input_control_vis/input_control_range.ts

```diff
@@ -49,9 +53,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     after(async () => {
       await esArchiver.unload('kibana_sample_data_flights_index_pattern');
       // loading back default data
+      await esArchiver.load('empty_kibana');
+
+      await kibanaServer.importExport.load('discover');
```

Comment:
Why is the loading 'discover' if it previously loaded 'visualize'?  copy/paste error?  I'm not sure it really needs to load anything in this after method.  The next test should either do the `init` call or load whatever data it needs.

Reply:
I think I was following the previously established pattern.  But that makes sense. Fixing up now.